### ISSUE TITLE
Whether to consider datasets in public papers？

### ISSUE THREAD

**Issue by lzchao-0917** [wind1117/AAAI24-PCDNet#2]
Hello, will you consider making the dataset public later? I am very interested in this polarization detection. Can you provide a dataset for research? Thank you very much.

**Comment by wind1117** [comment]
Of course, no problem. I apologize for my oversight; previously, I only included the dataset acquisition link on the project page. I have now updated the readme file, and I hope you can find the information you need.

**Comment by lzchao-0917** [comment]
> Of course, no problem. I apologize for my oversight; previously, I only included the dataset acquisition link on the project page. I have now updated the readme file, and I hope you can find the information you need.

Thank you for your selfless sharing！

**Comment by yugitw** [comment]
Hi thanks for sharing the data. Would you consider uploading data to a more open platform like OneDrive/Google Drive? We are having some issues accessing the BaiduNetDisk. 

Thanks!

**Comment by wind1117** [comment]
> Hi thanks for sharing the data. Would you consider uploading data to a more open platform like OneDrive/Google Drive? We are having some issues accessing the BaiduNetDisk.
> 
> Thanks!

Hello, the dataset is now on Google Drive. In a few days, you can apply for download again. Then you'll have both BaiduNetDisk and Google Drive download links. Hope this helps.
